libcloud/compute/drivers/azure_arm.py

```diff
@@ -2065,6 +2079,41 @@ def ex_create_tags(self, resource, tags, replace=False):
             method="PATCH",
         )
 
+    def ex_create_additional_capabilities(
+        self, node, additional_capabilities, resource_group,
+    ):
+        """
+        Set the additional capabilities on a stopped node.
+
+        :param node: The node to be updated
+        :type node: ``str``
+        :param ex_additional_capabilities: Optional additional capabilities
+            allowing Ultra SSD and hibernation on this node.
+        :type ex_additional_capabilities: ``dict``
+        :param resource_group: The resource group of the node to be updated
+        :type resource_group: ``str``
+        """
+
+        target = (
+            "/subscriptions/%s/resourceGroups/%s/providers"
+            "/Microsoft.Compute/virtualMachines/%s"
+            % (self.subscription_id, resource_group, node.name)
+        )
+
+        data = {
+            "location": node.extra["location"],
+            "properties": {
+                "additionalCapabilities": additional_capabilities
+            }
+        }
+
+        self.connection.request(
+            target,
+            data=data,
+            params={"api-version": VM_API_VERSION},
+            method="PUT",
+        )
+
```

Comment:
Does this API operation return some kind of response? Would it make sense to return something from this method (depending also on what API operation returns)? Perhaps True on success?

Reply:
Thanks for catching that. I updated the method as suggested.